File: Duran_TuranMert.php
<?php
$n = 10;
      echo("-----*-Pretest---While--*-------");
while($n > 0){
        echo("\r\n");
    if( $n == 7 ){
      $n--;
      echo ("-------*---continue at 7 *--------");
      continue;    
    }
    if( $n == 5 ){
      echo ("-------*---Break at 5 *--------");
      break;
    }
    echo $n ;
    $n--;
}
echo("\r\n");
echo("-----*-Pretest---For---*-------");
for ($i = 10; $i > 0; $i--) {
  echo("\r\n");
  if( $i == 7 ){
    echo ("-------*---continue at 7 *--------");
    continue;    
  }
  if( $i == 5 ){
    echo ("-------*---Break at 5 *--------");
    break;
  }
  echo $i ;
} 
      echo("\r\n");
      echo("-----*-Posttest-*-------");

$num = 10;
do{
    echo("\r\n");
    if( $num == 7 ){
      $num--;
      echo ("-------*---continue at 7 *--------");
      continue;    
    }
    if( $num == 5 ){
      echo ("-------*---Break at 5 *--------");
      break;
    }
    echo $num;
    $num--;
}
while($num > 0);
?>

File: Duran_TuranMert.rs
fn main() {
    println!("------*-Pretest-*-------");
    let mut num = 10;
    while num > 0{
       if num == 5 {
         println!("------*-Break at 5-*-------");
          break;
       }
       if num == 8 {
         num = num - 1;
         println!("------*-Continue at 8-*-------");
         continue;
       }
       
       println!("{}",num);
       num-=1;
    }
    println!("");
    println!("------*-Posttest-*-------");
    let mut n = 10;
    loop {
       if n == 5 {
          println!("------*-Break at 5-*-------");
          break;
       }
       if n == 8 {
         n = n - 1;
         println!("------*-Continue at 8-*-------");
         continue;
       }
       
       println!("{}",n);
       n-=1;
    }
 }